Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -288,16 +276,28 @@ final class Analytics {
         track(event)
     }
     
-    func introClick(_ label: Label.Navigation, at page: Property.OnboardingPage?) {
-        guard let page,
-              let pageAsInt = Property.OnboardingPage.allCases.firstIndex(of: page) else {
+    func introDisplaying(page: Property.OnboardingPage?, index: Int) {
```

Comment:
Given the utilization of this `index` value, I thought that might be closer to Apple's syntax utilizing the `at` in the function signature.
```suggestion
    func introDisplaying(page: Property.OnboardingPage?, at index: Int) {
```

Reply:
I like it! Will update 🙂